src/IR2Vec.cpp

```diff
@@ -113,8 +119,25 @@ int main(int argc, char **argv) {
     exit(1);
 
   auto M = getLLVMIR();
+  // newly added
```

Comment:
Seems like the file descriptors are not closed even in the existing code. Can you please close them appropriately?